js/ngDialog.js

```diff
@@ -544,7 +544,8 @@
                             }
 
                             if (options.width) {
-                                $dialog[0].querySelector('.ngdialog-content').style.width = options.width + 'px';
+                                var $dialogContent = $dialog[0].querySelector('.ngdialog-content');
+                                (angular.isString(options.width)) ? $dialogContent.style.width = options.width : $dialogContent.style.width = options.width + 'px';
```

Comment:
I would prefer this as an expanded conditional, as ternaries like this are hard for some of us (slow people like me) to understand without stopping to think.